apps/pie-docs/src/index.md

```diff
@@ -1,6 +1,11 @@
 ---
 layout: layouts/page-layout.njk
-title: Principles for Interfaces and Experiences
-description: PIE is the global design system for Just Eat Takeaway.com, powering all our products.
+title: PIE Design System
+description: Just Eat Takeaway.com’s global principles for interfaces and experiences
 heroImageSrc: home-hero.svg
+heroImageWidth: 424px
```

Comment:
the designs actually show this image scaling down as the screensize shrinks - might not be ideal to hard code the absolute size of it. Maybe a max width and an aspect ratio?

Reply:
@jamieomaguire @raoufswe It did look good with a fixed size for the image, but I would like to see it scaling down too if possible and make a decision based on the results?

Reply:
the image will scale dynamically now 💯